app/experimenter/experiments/api/v5/types.py

```diff
@@ -79,6 +79,7 @@ class NimbusCountryType(DjangoObjectType):
 
     class Meta:
         model = Country
+        fields = ("id", "code", "name")
```

Comment:
since we are adding all fields, how about `fields = "__all__"`

Reply:
Well, the pattern I was finding was that in a bunch of places we were doing essentially that but by using `exclude` and then fields were unintentionally showing up in the objects and it wasn't clear because there wasn't an explicit list of fields, so I started feeling like we should avoid that pattern and always explicitly list fields.

Reply:
that's a great point